pkg/crdutil/crdutil.go

```diff
@@ -49,26 +51,32 @@ func (s *StringList) Set(value string) error {
 }
 
 var (
-	crdsDir StringList
+	files     []string
+	recursive bool
 )
 
 func initFlags() {
-	flag.Var(&crdsDir, "crds-dir", "Path to the directory containing the CRD manifests")
-	flag.Parse()
-
-	if len(crdsDir) == 0 {
-		log.Fatalf("CRDs directory is required")
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+	pflag.StringSliceVarP(&files, "filename", "f", files,
+		"The files that contain the configurations to apply.")
+	pflag.BoolVarP(&recursive, "recursive", "R", false,
+		"Process the directory used in -f, --filename recursively. Useful when you want to manage related manifests organized within the same directory.")
```

Comment:
Is this not already handled without the `-R` flag? (This was the previous behaviour).